core/domain/story_services.py

```diff
@@ -402,20 +396,19 @@ def _save_story(committer_id, story, commit_message, change_list):
                 'but found an exploration with ID: %s (was it deleted?)' %
                 node.exploration_id)
 
-    story_model = story_models.StoryModel.get(story.id, strict=False)
-    if story_model is None:
-        story_model = story_models.StoryModel(id=story.id)
-    else:
-        if story.version > story_model.version:
-            raise Exception(
-                'Unexpected error: trying to update version %s of story '
-                'from version %s. Please reload the page and try again.'
-                % (story_model.version, story.version))
-        elif story.version < story_model.version:
-            raise Exception(
-                'Trying to update version %s of story from version %s, '
-                'which is too old. Please reload the page and try again.'
-                % (story_model.version, story.version))
+    # Story model cannot be None as story is passed as parameter here and that
```

Comment:
To clarify, is the argument here, also the fact that it's a private function and so should not be called externally?
If so, add that also in the comments. If not, we can call this with basically any story object that need not exist in the datastore.

Reply:
Done